Hearthstone Deck Tracker/Enums/ShowDateOnDecksOptions.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.ComponentModel;
+
+namespace Hearthstone_Deck_Tracker.Enums
+{
+	public enum ShowDateOnDecksOptions
+	{
+		[LocDescription("Enum_Show_No_Date")]
+		showNoDate,
```

Comment:
Should start with uppercase

Reply:
With renaming the enum to `DateType` this can just be `None`. (localization string should be renamed too)